Jenkinsfile

```diff
@@ -340,7 +349,7 @@ pipeline {
         stage('Publish-Image') {
             when {
                     anyOf {
-                        branch 'develop'
+                        branch 'developn'
```

Comment:
Is this intentional?

Reply:
Fixed in the latest push, that was there for Jenkins copy paste so it didn't go through the publish image step